pkg/controller/migmigration/task.go

```diff
@@ -474,6 +474,9 @@ func (t *Task) Run() error {
 		} else {
 			t.Requeue = PollReQ
 		}
+		if len(report.progress) > 0 {
```

Comment:
No need for a check, you can always just set an empty progress array and it will be omitted.